### PR TITLE
Enable touch-click interaction on mobile

### LINKED ISSUE #19: Mobile marker won't call onClick
If the users clicks on a marker on mobile the onClick event will not be fired if present on the marker.
This happens because the handler for onTouchEnd prevents the default behavior of the interaction.
If we check to see if the touch started and ended at about the same place we can tell it's a click and not a drag and stop preventing the default behavior and dragging the map.

### PR DESCRIPTION
**Pull request by degroote22**
fixes #19

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -390,14 +390,31 @@ export default class Map extends Component {
...
+        // if the click started and ended at about
+        // the same place we can view it as a click
+        // and not prevent default behavior.
+        const start = this._touchStartCoords[0]
+        const touch = event.changedTouches[0]
+        const end = [touch.clientX, touch.clientY]
+
+        if (
+          Math.abs(start[0] - end[0]) > 5
+          || Math.abs(start[1] - end[1]) > 5
```
Comment (mariusandra) [inline src/index.js]: could you put this `5` into a constant like the other ones at the top of the file?

Comment (degroote22) [inline src/index.js]: will do

**Comment by degroote22** [comment]
This only fixed the issue on the mobile emulating tool of chrome.
On a real device onClick isn't being called yet with this fix.
I'm sorry for the mess and I'll try to come up with a real solution.

**Comment by degroote22** [comment]
Now it works. But only if preventDefault isn't being called on onTouchStart.
IDK what happens if we just don't call it and haven't thought on a good heuristic to catch correct clicks.
But, for me, it's working like a charm.

**Comment by mariusandra** [comment]
Hey! After a slight 8 month delay, this PR is now merged! :tada:
`v0.9.2` is out with it!

**Pull request merged**